dump-format.mediawiki

```diff
@@ -0,0 +1,94 @@
+<pre>
+  NEP: TBD
+  Title: Unified Neo chain dump format
+  Author: Ekaterina Pavlova <ekt@nspcc.ru>, Roman Khimov <roman@nspcc.ru>
+  Type: Informational
+  Status: Draft
+  Created: 2025-02-14
+</pre>
+
+==Abstract==
+
+This proposal standardizes a single Neo chain dump format that unifies the “full” and “incremental”
+approaches into one consistent structure. It enables different Neo node implementations to remain
+compatible when producing or consuming chain dumps, allowing easy import or export of chain
+data without confusion over file naming or format variations.
+
+==Motivation==
+
+Neo nodes have historically used two chain dump formats: a full dump from block 0 (e.g., chain.acc)
+and an incremental dump from an arbitrary block (e.g., chain.X.acc). This dual approach complicates
+tooling and confuses operators (e.g., chain.0.acc is actually incremental). This NEP unifies both
+formats into one, making dumps interchangeable, simplifying import/restore, and reducing confusion.
+
+==Specification==
+
+===Dump Structure===
+
+Multibyte data items are always stored in little-endian order, where the low bytes come first.
+Its structure is described further below.
+
+<pre>
+dump_file
+{
+  4-byte start;
+  4-byte count;
```

Comment:
if we stream it, this can be -1?

Reply:
This file contains the restricted number of blocks anyway.